crates/ruff/src/importer/insertion.rs

```diff
@@ -313,7 +313,7 @@ mod tests {
     fn start_of_file() -> Result<()> {
         fn insert(contents: &str) -> Result<Insertion> {
             let program = Suite::parse(contents, "<filename>")?;
-            let tokens: Vec<LexResult> = ruff_rustpython::tokenize(contents);
+            let tokens: Vec<LexResult> = ruff_rustpython::tokenize(contents, Mode::Module);
```

Comment:
Unrelated to this line. Ruff calls into `lex_starts_at` in various places. Do we need to change these calls to use the right mode?

Reply:
Yes. Most of the calls are made in the rule implementation so they'll require the correct context.